src/dom/native/NativeViewElementNode.ts

```diff
@@ -259,12 +259,12 @@ export default class NativeViewElementNode<T extends View> extends NativeElement
         //childnodes with propAttributes aren't added to native views
         if (childNode.propAttribute) return;
 
-        if (!this.nativeView || !childNode.nativeView) {
+        const parentView = this.nativeView
+        const childView = childNode.nativeView
+        if (!parentView || !childView || !(childView instanceof View)) {
```

Comment:
I might change this to check against ViewBase to match the requirements of the methods below. (https://github.com/NativeScript/NativeScript/blob/66cc8a477e166cf4dbea33a06a49eea018bd8933/nativescript-core/ui/core/view-base/view-base.ts#L648)